src/app/lib/config/services/ukrainian.js

```diff
@@ -5,275 +5,410 @@ import '@bbc/moment-timezone-include/tz/GMT';
+    topStoriesTitle: 'Top Stories',
+    featuresAnalysisTitle: 'Докладно',
+  },
+  brandSVG,
+  mostRead: {
+    header: 'Найпопулярніше',
+    lastUpdated: 'Останнє оновлення:',
+    numberOfItems: 10,
+    hasMostRead: true,
+    onIdxPage: false,
+  },
+  radioSchedule: {
+    hasRadioSchedule: false,
+    onIdxPage: false,
+  },
+  recommendations: {
+    hasStoryRecommendations: false,
+  },
+  footer: {
+    trustProjectLink: {
+      href: 'https://www.bbc.com/ukrainian/institutional-50170368',
+      text: 'Чому BBC заслуговує на довіру',
+    },
+    externalLink: {
+      href: 'https://www.bbc.co.uk/help/web/links/',
+      text: 'Ознайомтеся з нашими правилами зовнішніх посилань.',
     },
-    fonts: [],
-    timezone: 'GMT',
-    navigation: [
+    links: [
       {
-        title: 'Новини',
-        url: '/ukrainian',
+        href: 'https://www.bbc.com/ukrainian/institutional-38144387',
+        text: 'Правила користування',
       },
       {
-        title: 'Коронавірус',
-        url: '/ukrainian/52118881',
+        href: 'https://www.bbc.com/ukrainian/institutional-38144827',
+        text: 'Особисті дані',
       },
       {
-        title: 'Наука',
-        url: '/ukrainian/topics/0f469e6a-d4a6-46f2-b727-2bd039cb6b53',
+        href: 'https://www.bbc.com/usingthebbc/cookies/',
+        text: 'Cookies',
       },
       {
-        title: 'Технології',
-        url: '/ukrainian/topics/31684f19-84d6-41f6-b033-7ae08098572a',
+        href: 'https://www.bbc.com/ukrainian/institutional-38146278',
+        text: "Зв'язатися з BBC",
       },
       {
-        title: 'Здоров’я',
-        url: '/ukrainian/topics/c4794229-7f87-43ce-ac0a-6cfcd6d3cef2',
+        href:
+          'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
+        text: 'AdChoices / Do Not Sell My Info',
+        lang: 'en-GB',
       },
-      {
-        title: 'Журнал',
-        url: '/ukrainian/magazine',
+    ],
+    copyrightText: 'BBC. ВВС не несе відповідальності за контент інших сайтів.',
+  },
+  fonts: [],
+  timezone: 'GMT',
+  navigation: [
+    {
+      title: 'Новини',
+      url: '/ukrainian',
+    },
+    {
+      title: 'Коронавірус',
+      url: '/ukrainian/52118881',
+    },
+    {
+      title: 'Наука',
+      url: '/ukrainian/topics/0f469e6a-d4a6-46f2-b727-2bd039cb6b53',
+    },
+    {
+      title: 'Технології',
+      url: '/ukrainian/topics/31684f19-84d6-41f6-b033-7ae08098572a',
+    },
+    {
+      title: 'Здоров’я',
+      url: '/ukrainian/topics/c4794229-7f87-43ce-ac0a-6cfcd6d3cef2',
+    },
+    {
+      title: 'Журнал',
+      url: '/ukrainian/magazine',
+    },
+    {
+      title: 'Книга року ВВС',
+      url: '/ukrainian/features-50320117',
+    },
+    {
+      title: 'Блоги',
+      url: '/ukrainian/blogs',
+    },
+    {
+      title: 'РУС',
+      url: '/ukrainian/ukraine_in_russian',
+    },
+  ],
+};
+
+const russianTranslations = {
+  translations: {
+    seeAll: 'khoa phan translations',
```

Comment:
😄

Reply:
For others reviewing, the ws-home team decided to use the translations from the russian.js service config for the mean time.